### ISSUE TITLE
VIVO-1663: Add more person identifiers to VIVO

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#3253]
[Don Elsborg](https://github.com/delsborg) (Migrated from [VIVO-1663](https://vivo-project.atlassian.net/browse/VIVO-1663)) said:

There is a desire by various institutions, like CU Boulder, to add more identifiers into VIVO.

Some of this was done in OpenVIVO - see the links under Weblinks at: [http://openvivo.org/display/orcid0000-0002-1304-8447]

CU Boulder is already adding the following in their 1.9.3 instance via data properties with range of anyURI:
 * Twitter
 * [LinkedIn?](https://wush.net/trac/fis/wiki/LinkedIn)
 * Google Scholar
 * [ResearchGate?](https://wush.net/trac/fis/wiki/ResearchGate)
 * Academia.edu
 * Figshare Identifier
 * [GitHub](https://wush.net/trac/fis/wiki/GitHub) Identifier

Ralph from UAB mentioned:
 * arXiv Author Identifier
ResearcherID
Scopus IDs
SSRN Author ID
Twitter
National Provider Identifier Standard (NPI)

**Comment by chenejac** [comment]
This issue is related to #2927

**Comment by hauschke** [comment]
Can we move this discussion to https://github.com/vivo-ontologies/vivo-ontology/issues/75 and close this issue, @chenejac?

**Comment by chenejac** [comment]
Yes, @hauschke is right, this should be resolved in the vivo-ontology repository.
